src/test/resources/features/personalInformation/edit_personalInformation.feature

```diff
@@ -0,0 +1,18 @@
+@PersonalInformation
+Feature: Edit Personal Information
+
+  Background:
+    Given I log in to Salesforce with a valid Account
+
+  @restorePersonalInformation
+  Scenario: Update personal information with the minimum values
+    When I go to settings
+    And I verify that the title of the panel is "Personal Information"
```

Comment:
It sounds like an assertion, in this case it should start with "Then" but, it's necesary to do this assertion?
I think that it will be better to change to "I go to Personal Information page"

Reply:
Okey